### DISCUSSION TITLE
Estimation Methods - Forecasts

### DISCUSSION THREAD

**Discussion by alixunderplatz** [electricitymaps/electricitymaps-contrib#3701] [Ideas] [answered]
Hi all, hi @madsnedergaard,
I've seen that you display estimated data for some of the zones with no live-data - great thing :)!
The "duplicator model" you describe is a good method to start with.

Are you guys using any of the official forecasts yet? Most of the parsed data sources of TSOs or market authorities seem to provide such day-ahead estimates.

For ENTSO-E, as you know, there is no split of the forecasted production, just a total generation forecast and separate solar and wind forecasts. However, using the forecasts for solar and onshore/offshore wind, you can determine the two most influential sources, when it comes to change of CO2-intensity.

**Comment by madsnedergaard** [comment, marked as answer]
Hey @alixunderplatz, thanks for opening this discussion.

We're currently working internally on a proper estimation model, where the duplicator was thought as a very basic, temporary step to get it working on the frontend. We're hoping to roll the new estimation model out during this quarter, although there might always be unknown unknowns :)

I can't recall if we use the day-ahead estimates as a feature for the new models... @pierresegonne?

**Reply by pierresegonne** [reply]
Hey @alixunderplatz, thanks for the kind words :) 

Glad to see that people are appreciating seeing the estimates ;)

For our official forecasts, that are part of the API, we do include these forecasts when available into the set of features used. For the estimation models that we will roll-out soon(ish), we do not use these as of yet. This is mainly because we chose to first focus on setting up the infrastructure right, and improve the models later on :) So we'll keep your suggestion in mind when we improve on these!

**Reply by alixunderplatz** [reply]
I guess the power flows are deactivated for zones with estimated data, as we can see for Belgium and Germany at the moment, since
you don't want to mess up the carbon intensity / carbon flows?
![grafik](https://user-images.githubusercontent.com/25743609/151310692-a98003b9-8707-4a1a-b365-7ad15380c1b3.png)

**Reply by madsnedergaard** [reply]
> I guess the power flows are deactivated for zones with estimated data, as we can see for Belgium and Germany at the moment, since you don't want to mess up the carbon intensity / carbon flows?

Yes exactly! ;)

**Comment by alixunderplatz** [comment]
Wow, these new estimates are terrific!!! 👍  👍  👍 
Production-profiles of Korea, Japan-Kansai and Japan-Tokyo looking really great! Well done!

![grafik](https://user-images.githubusercontent.com/25743609/153726523-a4518b73-6b07-4fb3-9447-4b85a4be8d88.png)

**Reply by madsnedergaard** [reply]
Thanks a lot from the entire team! 😍 
And kudos for these profiles goes to @gwpicard 🚀
